.github/workflows/slowtests.yml

```diff
@@ -7,20 +7,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out repository code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: Setup Java environment
         uses: actions/setup-java@v1
         with:
-          java-version: '17'
+          java-version: 21
```

Comment:
Tipper fnuttene er tatt vekk med vilje?

Reply:
Ja, bare for å gjøre det likt. De har ingen hensikt.